www/js/lib/uiUtil.js

```diff
@@ -406,9 +406,9 @@ define(rqDef, function(settingsStore) {
      * Removes the animation effect between various sections
      */
     function removeAnimationClasses() {
-        $('#about').removeClass('slideIn_L').removeClass('slideOut_R');
-        $('#configuration').removeClass('slideIn_L').removeClass('slideIn_R').removeClass('slideOut_L').removeClass('slideOut_R');
-        $('#articleContent').removeClass('slideIn_R').removeClass('slideOut_L');
+        document.getElementById('about').classList.remove('slideIn_L','slideOut_R');
```

Comment:
Add a space after the comma, for consistency.

Reply:
Sure.